packages/garbo/src/tasks/barfTurn.ts

```diff
@@ -260,6 +261,18 @@ function willDrunkAdventure() {
   return have($item`Drunkula's wineglass`) && globalOptions.ascend;
 }
 
+function canGetFusedFuse() {
+  if (get("noncombatForcerActive")) return true;
```

Comment:
the return true here doesn't check for the realm being available or the fused fuse being available in _volcanoItem#